ralph/status.sh

```
@@ -0,0 +1,161 @@
+  # Count stories
+  TOTAL=$(jq '.userStories | length' "$PRD" 2>/dev/null || echo 0)
+  TOTAL=${TOTAL:-0}
+  if [ "$TOTAL" -eq 0 ]; then
+    PASSED=0
+    FAILED=0
+    PCT=0
+    FILLED=0
+    EMPTY=40
+  else
+    PASSED=$(jq '[.userStories[] | select(.passes == true)] | length' "$PRD")
+    FAILED=$((TOTAL - PASSED))
```

Shellcheck (0.11.0)

[warning] FAILED appears unused. Verify use (or export if used externally).

(SC2034)